executor/join.go

```diff
@@ -402,13 +401,13 @@ func (e *HashJoinExec) joinMatchedOuterRow2Chunk(workerID uint, outerRow chunk.R
 		return false, joinResult
 	}
 	if hasNull {
-		e.joiners[workerID].onMissMatch(outerRow, joinResult.chk)
+		e.joiners[workerID].onMissMatch(false, outerRow, joinResult.chk)
```

Comment:
should be `true`?

Reply:
Here we are evaluating if the outer key of a normal column equal condition is null, not evaluating the `OtherConditions`, i.e, conditions converted from `in`, so we should use `false` instead of `true`.